portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,13 +78,31 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     response.getWriter().print("A message from: ");
     response.getWriter().println(Arrays.toString(names));
     response.getWriter().println(Arrays.toString(words));
-  }
 
+    // Here starts the New Comment part.
+    String title = request.getParameter("title");
+    long timestamp = System.currentTimeMillis();
+
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("title", title);
+    commentEntity.setProperty("timestamp", timestamp);
+    
+    // Create a server connection to get data and put new comments to teh database.
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(commentEntity);
+
+    // After the procedure go back to the home page.
+    response.sendRedirect("/index.html");
+
+  }
+  
   /**
    * @return the request parameter, or the default value if the parameter
    *         was not specified by the client
    */
-  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
+  
+  private String getParameter(HttpServletRequest request, String name, String defaultValue) 
```

Comment:
Can you add @param javadoc comments for this function, sorry I didn't catch this last review)

Reply:
done, added